src/test/java/com/github/dockerjava/cmd/LoadImageCmdIT.java

```diff
@@ -41,6 +41,11 @@ public void loadImageFromTar() throws Exception {
             dockerRule.getClient().loadImageCmd(uploadStream).exec();
         }
 
+        //swarm needs some time to refelct new images
+        synchronized (this) {
+            wait(5000);
```

Comment:
what is the criteria that it swarm is ready?

Reply:
I don't know but we use the same trick in our production warm cluster. I guess that swarm manager keeps an image cache in memory or something like that.
And this cache is lazy reinitialized after a new image is in place.

Reply:
I don't know but we use the same trick in our production swarm cluster. I guess that swarm manager keeps an image cache in memory or something like that.
And this cache is lazy reinitialized after a new image is in place.